Content.Server/NukeOps/WarDeclaratorComponent.cs

```diff
@@ -22,10 +22,6 @@ public sealed partial class WarDeclaratorComponent : Component
     [DataField]
     public bool AllowEditingMessage = true;
 
-    [ViewVariables(VVAccess.ReadWrite)]
-    [DataField]
-    public int MaxMessageLength = 512;
-
```

Comment:
The component's field was never used really because of the hardcoded `.Substring(0, 256)` in the WarDeclaratorSystem. I think it's better to have it configured consistently via a cvar, than to allow components to have it. Not to mention that the component is scoped to the Content.Server namespace, and it's impossible to make use of it on the Client side.